server/src/infra/repositories/asset.repository.ts

```diff
@@ -133,6 +138,58 @@ export class AssetRepository implements IAssetRepository {
     });
   }
 
+  async addExtractedLivePhoto(
```

Comment:
Can you move this implementation to the metadata processor?

Reply:
Done.